src/main/java/com/adobe/target/edge/client/ondevice/collator/UserParamsCollator.java

```diff
@@ -101,24 +101,39 @@
     Map<String, Object> user = new HashMap<>();
     String userAgent = extractUserAgent(deliveryRequest);
     ClientHints clientHints = extractClientHints(deliveryRequest);
-    user.put(USER_BROWSER_TYPE, parseBrowserType(userAgent));
+    String browserInfo = getBrowserInfo(userAgent, clientHints);
+    user.put(USER_BROWSER_TYPE, parseBrowserType(browserInfo));
```

Comment:
Let's just save the result of `parseBrowserType(browserInfo)` to a variable so we can more easily use it in the `parseBrowserVersion` call.